napari/plugins/npe2api.py

```diff
@@ -71,8 +92,14 @@ def conda_map() -> Dict[PyPIname, Optional[str]]:
         return json.load(resp)
 
 
+class _ExtraDict(TypedDict):
+    home_page: str
+    pypi_versions: List[str]
+    conda_versions: List[str]
+
+
 def iter_napari_plugin_info() -> (
-    Iterator[Tuple[PackageMetadata, bool, Dict[str, str]]]
+    Iterator[Tuple[PackageMetadata, bool, _ExtraDict]]
```

Comment:
As above. I find the TypedDict subclassing quite unreadable and unpythonic; it adds a layer of "nonlocality" to the type annotations. Even if here it is immediately above the signature, that won't always be true or obvious.

Reply:
I see. From one side TypedDict is unpythonic. On other side allows finding errors without writing tests.